apps/studio/components/interfaces/Storage/AnalyticsBuckets/CreateAnalyticsBucketModal.tsx

```diff
@@ -195,7 +195,6 @@ export const CreateAnalyticsBucketModal = ({
           className={buttonClassName}
           icon={<Plus size={14} />}
           disabled={isDisabled}
-          tabIndex={isDisabled ? -1 : 0}
```

Comment:
Now handled in the underlying `Button` component, which (by default) sets the `tabIndex` in the same way; based on the `disabled` state.